src/OpenDiffix.CLI/Program.fs

```diff
@@ -39,6 +40,8 @@ type CliArguments =
         + "The db_path inside should be relative to this file's path."
       | Query_Stdin -> "Reads the query from standard in."
       | Salt _ -> "The salt value to use when anonymizing the data. Changing the salt will change the result."
+      | Mode _ ->
+        "Anonymization mode: 'untrusted' mode is more restrictive and assumes secure anonymization despite the analyst actively attempting to de-anonymize the results."
```

Comment:
Exceeds line length, maybe split or shorten. Something like `'untrusted' assumes a malicious analyst actively attempting to de-anonymize the results`

Reply:
yup, there's a nice and short definition in the spec, I'll use this